pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   async: ^2.5.0
   crypto: ^3.0.0
   fixnum: ^1.0.0
-  googleapis_auth: ^1.1.0
+  googleapis_auth: ">=1.0.0 <3.0.0"
```

Comment:
Why leave the dependency on the old `googleapis_auth`?

Reply:
To let folks upgrade. There are no breaking changes as far as this package is concerned.